lib/modules/manager/nix/types.ts

```diff
@@ -0,0 +1,40 @@
+export interface NixFlakeLock {
```

Comment:
use zod schema

Reply:
that's complicated to parse a json file 💀

Reply:
Got it working with the help of a friend 😅